cmd/buck/cli/cli.go

```diff
@@ -1,100 +1,125 @@
-	bucketInitCmd.PersistentFlags().String("org", "", "Org username")
-	bucketInitCmd.PersistentFlags().String("thread", "", "Thread ID")
-	if err := cmd.BindFlags(config.Viper, bucketInitCmd, config.Flags); err != nil {
-		cmd.Fatal(err)
-	}
-	bucketInitCmd.Flags().StringP("name", "n", "", "Bucket name")
-	bucketInitCmd.Flags().BoolP("private", "p", false, "Obfuscates files and folders with encryption")
-	bucketInitCmd.Flags().String("cid", "", "Bootstrap the bucket with a UnixFS Cid from the IPFS network")
-	bucketInitCmd.Flags().BoolP("existing", "e", false, "Initializes from an existing remote bucket if true")
+	initCmd.PersistentFlags().String("key", "", "Bucket key")
+	initCmd.PersistentFlags().String("thread", "", "Thread ID")
+	initCmd.PersistentFlags().String("org", "", "Org username")
+
+	initCmd.Flags().StringP("name", "n", "", "Bucket name")
+	initCmd.Flags().BoolP("private", "p", false, "Obfuscates files and folders with encryption")
+	initCmd.Flags().String("cid", "", "Bootstrap the bucket with a UnixFS Cid from the IPFS network")
+	initCmd.Flags().BoolP("existing", "e", false, "Initializes from an existing remote bucket if true")
+
+	pushCmd.Flags().BoolP("force", "f", false, "Allows non-fast-forward updates if true")
+	pushCmd.Flags().BoolP("yes", "y", false, "Skips the confirmation prompt if true")
+	pushCmd.Flags().Int64("maxsize", buckMaxSizeMiB, "Max bucket size in MiB")
+
+	pullCmd.Flags().BoolP("force", "f", false, "Force pull all remote files if true")
+	pullCmd.Flags().Bool("hard", false, "Pulls and prunes local changes if true")
+	pullCmd.Flags().BoolP("yes", "y", false, "Skips the confirmation prompt if true")
+
+	addCmd.Flags().BoolP("yes", "y", false, "Skips confirmations prompts to always overwrite files and merge folders")
 
-	bucketPushCmd.Flags().BoolP("force", "f", false, "Allows non-fast-forward updates if true")
-	bucketPushCmd.Flags().BoolP("yes", "y", false, "Skips the confirmation prompt if true")
-	bucketPushCmd.Flags().Int64("maxsize", buckMaxSizeMiB, "Max bucket size in MiB")
+	watchCmd.Flags().Duration("interval", time.Millisecond*100, "The interval at which local changes are synced remotely")
 
-	bucketPullCmd.Flags().BoolP("force", "f", false, "Force pull all remote files if true")
-	bucketPullCmd.Flags().Bool("hard", false, "Pulls and prunes local changes if true")
-	bucketPullCmd.Flags().BoolP("yes", "y", false, "Skips the confirmation prompt if true")
+	encryptCmd.Flags().StringP("password", "p", "", "Encryption password")
+	decryptCmd.Flags().StringP("password", "p", "", "Decryption password")
 
-	bucketAddCmd.Flags().BoolP("yes", "y", false, "Skips confirmations prompts to always overwrite files and merge folders")
+	archiveStatusCmd.Flags().BoolP("watch", "w", false, "Watch execution log")
+}
 
-	bucketEncryptCmd.Flags().StringP("password", "p", "", "Encryption password")
-	bucketDecryptCmd.Flags().StringP("password", "p", "", "Decryption password")
+func SetBucks(b *local.Buckets) {
+	bucks = b
```

Comment:
Instead of setting the `clients` as before, we set the main `Buckets` object, which contains clients and a configuration config.